site/examples/inputs.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const inputWithButton = `<fieldset class='relative'>
+  <button class='btn absolute top right w96 round-r'>Search</button>
+  <input class='input pr96' placeholder='Search'>
```

Comment:
Could this example be rewritten with input that has: `border--r--0 round-left` and is positioned beside a button, with no absolute positioning? I think that would be a cleaner implementation.